pkg/solana/contract.go

```diff
@@ -168,13 +198,27 @@ func GetLatestTransmission(ctx context.Context, client *rpc.Client, account sola
 		return Answer{}, 0, errTransmissionLength
 	}
 
+	var timestamp uint64
+	raw := make([]byte, 16)
+
+	buf = bytes.NewReader(t)
+	err = binary.Read(buf, binary.LittleEndian, &timestamp)
+	if err != nil {
+		return Answer{}, 0, err
+	}
+
+	// TODO: we could use ag_binary.Int128 instead
+	err = binary.Read(buf, binary.LittleEndian, &raw)
+	if err != nil {
+		return Answer{}, 0, err
+	}
```

Comment:
We should be able to just use `ag_binary` to decode these into a struct rather than manual bit fiddling.